src/infinite.ts

```diff
@@ -0,0 +1,78 @@
+import type { Client } from "openapi-fetch";
+import type { MediaType, PathsWithMethod } from "openapi-typescript-helpers";
+import useSWRInfinite, {
```

Comment:
Is infinite maybe worth putting it it's own slash path the same way SWR does? I ask because it seems like part of their [v1 release did this with intention](https://swr.vercel.app/blog/swr-v1.en-US#smaller-size)

Maybe not necessary just a thought since it is what they did

Reply:
I actually spent quite a while on trying to implement that, but ESM-only tree-shaking was much simpler to rely on. There are some strange hacks going on in swr (look in [`swr/dist/infinite`](https://www.npmjs.com/package/swr?activeTab=code)) that support their bundling, and I couldn't get it to work in consumers even when copying their methodology.